Sources/MapLibreSwiftUI/Models/MapCamera/MapViewCamera.swift

```diff
@@ -32,25 +47,76 @@ public struct MapViewCamera {
     /// - Returns: The constructed MapViewCamera.
     public static func center(_ coordinate: CLLocationCoordinate2D,
                               zoom: Double,
-                              pitch: Double = 90.0,
-                              course: Double = 0) -> MapViewCamera {
+                              pitch: Double? = Defaults.pitch,
+                              course: Double = Defaults.course,
+                              reason: CameraChangeReason? = nil) -> MapViewCamera {
         
         return MapViewCamera(state: .centered,
                              coordinate: coordinate,
                              zoom: zoom,
                              pitch: pitch,
-                             course: course)
+                             course: course,
+                             lastReasonForChange: reason)
     }
     
-    public static func trackUserLocation(_ location: CLLocation,
-                                         zoom: Double,
-                                         pitch: Double = 90.0) -> MapViewCamera {
+    /// Enables user location tracking within the MapView.
+    ///
+    /// This feature uses the MLNMapView's userTrackingMode = .follow
+    ///
+    /// - Parameters:
+    ///   - zoom: Set the desired zoom. This is a one time event and the user can manipulate their zoom after unlike pitch.
+    ///   - pitch: Provide a fixed pitch value. The user will not be able to adjust pitch using gestures when this is set. Use nil/default to allow user control.
+    /// - Returns: The MapViewCamera representing the scenario
+    public static func trackUserLocation(zoom: Double = Defaults.zoom,
+                                         pitch: Double? = Defaults.pitch) -> MapViewCamera {
         
+        // Coordinate is ignored when tracking user location. However, pitch and zoom are valid.
         return MapViewCamera(state: .trackingUserLocation,
-                             coordinate: location.coordinate,
+                             coordinate: Defaults.coordinate,
+                             zoom: zoom,
+                             pitch: pitch,
+                             course: Defaults.course,
+                             lastReasonForChange: .programmatic)
+    }
+    
+    /// Enables user location tracking within the MapView.
+    ///
+    /// This feature uses the MLNMapView's userTrackingMode = .follow
+    ///
+    /// - Parameters:
+    ///   - zoom: Set the desired zoom. This is a one time event and the user can manipulate their zoom after unlike pitch.
+    ///   - pitch: Provide a fixed pitch value. The user will not be able to adjust pitch using gestures when this is set. Use nil/default to allow user control.
```

Comment:
Dropping a note to update docs if we go for an enum rather than optional type.

Aside for my own clarification / discussion: Are gesture handlers the mechanism for "breaking out" of this when the user pans the map? If so, we'll probably also want start _and_ end gestures to support different interaction modes.